run/GCHP/createRunDir.sh

```diff
@@ -545,16 +498,15 @@ RUNDIR_VARS+="$(cat ${gcdir}/run/shared/settings/gmao_hemco.txt)\n"
 rundir_config_log=${rundir_config}/rundir_vars.txt
 echo -e "$RUNDIR_VARS" > ${rundir_config_log}
 
-# Call init_rd.sh
+# Initialize run directory
 ${srcrundir}/init_rd.sh ${rundir_config_log}
 
 #--------------------------------------------------------------------
 # Print run direcory setup info to screen
 #--------------------------------------------------------------------
 
-printf "\n  -- This run directory has been set up for $startdate - $enddate.\n"
-printf "\n  -- The default frequency and duration of diagnostics is set to monthly.\n"
-printf "\n  -- You may modify these settings in runConfig.sh.\n"
+printf "\n  -- This run directory has been set up to start on $start_date. Edit this in file cap_restart.\n"
+printf "\n  -- Edit other commonly changed run settings in setCommonRunSettings.sh, geoschem_config.yml, and HEMCO_Config.rc.\n"
```

Comment:
@lizziel Super minor thing - the new run directory setup info doesn't match the line width of the rest of the script, which comes out a little jarring when running it.

Reply:
Good catch. I'll push a fix to that.